pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -805,8 +805,8 @@ func getPidsForProcess(name, pidFile string) ([]int, error) {
 	}
 
 	// Try to lookup pid by process name
-	pids, err2 := procfs.PidOf(name)
-	if err2 == nil {
+	pids, err := procfs.PidOf(name)
+	if err == nil {
```

Comment:
This stuff is tricky :) Re the comment on line 813, don't we want to log/return the error from `getPidFromFile`? When we assign `err` here, we overwrite that value. 

I think, if `err` is defined inline, than its online available in the scope of the `if` statement - see https://www.calhoun.io/one-liner-if-statements-with-errors/, particularly "This is also why we can have an existing variable named err and then redeclare it inside of an if statement. The new variable is in a new scope, so it is permitted by the compiler.".

So I think we want to use the exact code @derekwaynecarr linked :)